storybookjs/storybook
# Docs / Vue - Getting Started - Examples

Issue:

## What I did
Update Vue Documentation With Examples for 
- Whats A Story / Vue Button Story With Args
- Setup / Vue - Your Component
- Setup / Vue - Preview Global Decorator

## How to test

- Is this testable with Jest or Chromatic screenshots? **No.**
- Does this need a new example in the kitchen sink apps? **No.**
- Does this need an update to the documentation? **Yes.**

If your answer is yes to any of these, please make sure to include it in your PR.
